xml/System.Windows.Forms/ListView.xml

```diff
@@ -2485,9 +2485,22 @@
         <ReturnType>System.Windows.Forms.ImageList</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>The currently set GroupIcon image list.</summary>
```

Comment:
I don't see `GroupIcon` reflected in the DocId of this API. I see `ImageList`. Is this description accurate?